tests/integration.rs

```diff
@@ -5854,6 +6001,86 @@ mod tests {
 
             handle_child_output(r, &output);
         }
+
+        #[test]
+        #[cfg(not(feature = "mshv"))]
+        fn test_windows_guest_disk_hotplug() {
+            let windows_guest = WindowsGuest::new();
+
+            let mut ovmf_path = dirs::home_dir().unwrap();
+            ovmf_path.push("workloads");
+            ovmf_path.push(OVMF_NAME);
+
+            let tmp_dir = TempDir::new_with_prefix("/tmp/ch").unwrap();
+            let api_socket = temp_api_path(&tmp_dir);
+
+            let mut child = GuestCommand::new(windows_guest.guest())
+                .args(&["--api-socket", &api_socket])
+                .args(&["--cpus", "boot=2,kvm_hyperv=on"])
+                .args(&["--memory", "size=2G,hotplug_size=5G"])
```

Comment:
Please remove the `hotplug_size=5G` since this test is not about memory hotplug. Just use `size=4G`.
This needs to be fixed in `test_windows_guest_netdev_hotplug` as well.

Reply:
Done in both places, so it should suffice for now.

Thansk.

Reply:
But actually, it might be better to use an opposite approach. Namely, to have a universal command in all tests, that would combine any possible option permutations. That way it would add an additional security those options combined together don't cause regressions. Generally, seems both ways would be useful and an option too much should be of no harm.

Thanks